app/src/main/java/com/mtesitoo/helper/ImageHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.mtesitoo.helper;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.support.v4.graphics.drawable.RoundedBitmapDrawable;
+import android.support.v4.graphics.drawable.RoundedBitmapDrawableFactory;
+
+/**
+ * Created by eduardodiaz on 04/09/2017.
+ */
+
+public class ImageHelper {
```

Comment:
Where is this class being used ?

Reply:
it's being used in ProfileFragment, around line 112, inside a Picasso library callback for creating a rounded image with border. I extracted this functionality to this new helper class in case it's needed elsewhere